controllers/gatekeeper_controller_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"testing"
+
+	operatorv1alpha1 "github.com/font/gatekeeper-operator/api/v1alpha1"
+	"github.com/openshift/library-go/pkg/manifest"
+	"github.com/stretchr/testify/assert"
```

Comment:
Any reason not to use the gomega package assertions https://onsi.github.io/gomega/#making-assertions that are already set up by the operator-sdk?

Reply:
@font no reason... don't know how i missed that. will make the switch now

Reply:
@font its been switched to gomega